src/app/service/terra-base.service.ts

```diff
@@ -34,31 +34,31 @@ export class TerraBaseService
     // TODO use D instead of any
     protected dataModel:{ [dataId:number]:any } = {};
 
-    private _alert:TerraAlertComponent = TerraAlertComponent.getInstance();
+    private alert:TerraAlertComponent = TerraAlertComponent.getInstance();
 
-    constructor(private _terraLoadingSpinnerService:TerraLoadingSpinnerService,
-                private _baseHttp:Http,
-                private _baseUrl:string,
-                private _isPlugin?:boolean)
+    constructor(private terraLoadingSpinnerService:TerraLoadingSpinnerService,
+                private baseHttp:Http,
```

Comment:
src/app/components/forms/dynamic-form/service/terra-dynamic-form.service.ts(11,14): error TS2415: Class 'TerraDynamicFormService' incorrectly extends base class 'TerraBaseService'.
  Types have separate declarations of a private property 'baseHttp'.